Sming/Core/Network/NtpClient.cpp

```diff
@@ -132,28 +132,34 @@ void NtpClient::onReceive(pbuf* buf, IpAddress remoteIP, uint16_t remotePort)
 	uint8_t ver = (versionMode & 0b00111000) >> 3;
 	uint8_t mode = (versionMode & 0x07);
 
-	if(mode == NTP_MODE_SERVER && (ver == NTP_VERSION || ver == (NTP_VERSION - 1))) {
-		//Most likely a correct NTP packet received.
+	if(mode != NTP_MODE_SERVER || (ver != NTP_VERSION && ver != (NTP_VERSION - 1))) {
+		// Got a dodgy packet so treat it as we would a response failure
```

Comment:
I like the "reflowing" of the code. Can you please change this original comment `// Got a dodgy packet so treat it as we would a response failure` to `// Got a dodgy packet - ignore it and retry` ?